tiny_dnn/util/nms.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (c) 2013, Taiga Nomi and the respective contributors
+  All rights reserved.
+
+  Use of this source code is governed by a BSD-style license that can be found
+  in the LICENSE file.
+*/
+#pragma once
+
+#include <vector>
+
+namespace tiny_dnn {
+
+struct bounding_box {
+  float_t x_min;
+  float_t y_min;
+  float_t x_max;
+  float_t y_max;
+  float_t score;
+
+  inline float_t area() const { return (x_max - x_min) * (y_max - y_min); }
+};
+
+inline float_t iou(const bounding_box &b1, const bounding_box &b2) {
+  float_t x1           = std::max(b1.x_min, b2.x_min);
+  float_t y1           = std::max(b1.y_min, b2.y_min);
+  float_t x2           = std::min(b1.x_max, b2.x_max);
+  float_t y2           = std::min(b1.y_max, b2.y_max);
+  float_t width        = std::max(0.0f, (x2 - x1 + 1));
+  float_t height       = std::max(0.0f, (y2 - y1 + 1));
```

Comment:
[Travis-ci job](https://travis-ci.org/tiny-dnn/tiny-dnn/jobs/443529189#L1184) failed due to the float suffix `f`.

I think you can overcome the problem by rewriting in this way.
```cpp
  float_t width        = std::max<float_t>(0.0, (x2 - x1 + 1));
  float_t height       = std::max<float_t>(0.0, (y2 - y1 + 1));
```